### ISSUE TITLE
Better methods, lower time complexity or space complexity

### ISSUE THREAD

**Issue by Dylanmxh** [Mzzopublic/C#54]
Is there a relatively better method to answer certain questions, which results in lower time complexity or space complexity?

**Comment by abheydhya** [comment]
Yep, choosing the right algorithm improve time and space efficiency. Example:- merge sort can sort a list with much better time complexity than selection sort (though some compromise is done on space).
